src/test/scala/edu/ie3/simona/model/thermal/ThermalHouseSpec.scala

```diff
@@ -56,23 +56,19 @@ class ThermalHouseSpec extends UnitSpec with HpInputTestData {
       val thermalHouseTest = thermalHouse(18, 22)
       val innerTemperature = Temperature(20, Celsius)
 
-      val thermalEnergyGain =
-        thermalHouseTest.calcThermalEnergyGain(Kilowatts(100), Seconds(3600))
-      val thermalEnergyLoss = thermalHouseTest.calcThermalEnergyLoss(
-        innerTemperature,
-        Temperature(10, Celsius),
-        Seconds(3600),
-      )
-      val thermalEnergyChange = thermalHouseTest.calcThermalEnergyChange(
-        thermalEnergyGain,
-        thermalEnergyLoss,
-      )
+      val thermalEnergyGain = KilowattHours(100)
+      val thermalEnergyLoss =
+        thermalHouseTest.ethLosses.calcThermalEnergyChange(
+          innerTemperature,
+          Temperature(10, Celsius),
+          Seconds(3600),
+        )
+      val thermalEnergyChange = thermalEnergyGain - thermalEnergyLoss
+
       val innerTemperatureChange =
-        thermalHouseTest.calcInnerTemperatureChange(thermalEnergyChange)
-      val newInnerTemperature = thermalHouseTest.calcNewInnerTemperature(
-        innerTemperature,
-        innerTemperatureChange,
-      )
+        thermalEnergyChange / thermalHouseTest.ethCapa
+
+      val newInnerTemperature = innerTemperature + innerTemperatureChange
```

Comment:
This test can just be removed alltogether. The methods that were formerly tested don't exist anymore.